x-pack/plugins/lens/public/shared_components/ignore_global_filter/settings_control.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFormRow, EuiSwitch } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+export function IgnoreGlobalFilterRowControl({
+  checked,
+  onChange,
+}: {
+  checked: boolean;
+  onChange: (newValue: boolean) => void;
+}) {
+  return (
+    <EuiFormRow
+      display="columnCompressedSwitch"
+      label={i18n.translate('xpack.lens.layerSettings.ignoreGlobalFilters', {
+        defaultMessage: 'Use global filters',
```

Comment:
Do we already use "global filters" naming in Kibana? Maybe it would be less confusing to say something like "Use search bar filters".

Reply:
We are using the same terminology in TSVB (it supports this feature for many versions) so we used the same term for the TSVB to Lens users to easily identify it.

@gchaps I don't know if you have any proposal here to improve the text. We can still merge I think and we can change the text in a followup PR is we decide in favor of it.

Reply:
Also the same terminology has been around in Lens for already few minor within the Annotation and Dashboard context.